beacon-chain/blockchain/service.go

```diff
@@ -364,15 +312,3 @@ func (c *ChainService) isBlockReadyForProcessing(block *pb.BeaconBlock, beaconSt
 	}
 	return nil
 }
-
-// sendAndDeleteCachedBlocks checks if there is any block saved in the cache with a
-// slot number equivalent to the current slot. If there is then the block is
-// sent to the incoming block channel and deleted from the cache.
-func (c *ChainService) sendAndDeleteCachedBlocks(currentSlot uint64, beaconState *pb.BeaconState) {
```

Comment:
No longer needed because now we process all blocks as they come, even if they have slots from the future as that would mean there are skipped slots